zetaclient/orchestrator/bootstrap.go

```diff
@@ -216,6 +218,11 @@ func (oc *Orchestrator) bootstrapSui(ctx context.Context, chain zctx.Chain) (*su
 
 	observer := suiobserver.New(baseObserver, suiClient, gateway)
 
+	// migrate inbound cursor
+	if err = observer.MigrateInboundCursorToV35(); err != nil {
```

Comment:
orchestrator bootstrap is supposed to be unopiniated

Logic related to migrate from upgrade should be clearly explained in the comment and explained, why it is needed.
It is not v35 but v36, we don't upgrade this right at v36 so the name should be different, mentioning the authenticated call change
We should also add a to track removing this from the codebase after the upgrade

Reply:
renamed in commit https://github.com/zeta-chain/node/pull/4138/commits/a49f613fce8b5775b0279ff52b79cedfed533980